packages/kit/src/exports/vite/index.js

```diff
@@ -572,7 +581,7 @@ function kit({ svelte_config }) {
 							preserveEntrySignatures: 'strict'
 						},
 						ssrEmitAssets: true,
-						target: ssr ? 'node16.14' : undefined
+						target: ssr ? 'node18.13' : 'es2022'
```

Comment:
Is this a drive-by adjustment?

Reply:
Sort of, but the `es2022` part is necessary for this PR because of the top level await thing. I guess we need to mention this in the migration notes